G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs

```diff
@@ -0,0 +1,34 @@
+﻿using G1ANT.Addon.UI.Api;
+using G1ANT.Addon.UI.Structures;
+using G1ANT.Language;
+
+namespace G1ANT.Addon.UI.Commands
+{
+    [Command(Name = "ui.getcheckboxstate",
+        Tooltip = "This command returns targeted checkbox's state")]
+    public class UIGetCheckboxStateCommand : Command
+    {
+        public class Arguments : CommandArguments
+        {
+            [Argument(Required = true, Tooltip = "Desktop application checkbox")]
```

Comment:
I would suggest something like "WPath to checkbox" or something more related to what this argument is all about